layers/+spacemacs/spacemacs-editing/packages.el

```diff
@@ -31,6 +31,8 @@
         string-inflection
         undo-tree
         uuidgen
+        (vimish-fold :toggle (eq 'vimish dotspacemacs-folding-method))
+        (evil-vimish-fold :toggle (eq 'vimish dotspacemacs-folding-method))
```

Comment:
use `when`.

Reply:
https://www.gnu.org/software/emacs/manual/html_node/elisp/Conditionals.html

Reply:
@robbyoconnor like this?

```elisp
(when (eq 'vimish dotspacemacs-folding-method) vimish-fold evil-vimish-fold)
```

Hm, that can't be quite right.

Reply:
Do I need to use a quasiquote as well?

```elisp
`(
  ...
  @,(when (eq 'vimish dotspacemacs-folding-method) '(vimish-fold evil-vimish-fold))
  ...
  )
```

Reply:
Check other layers

Reply:
I don't think we need a `when` statement here, as `when` and `if` take a condition and then either execute an action or choose between two alternative actions. In this case this is already covered by the `:toggle` label which will take the condition.